### ISSUE TITLE
Add email assignment to assignee option to ADMIN - WORK ASSIGNMENT FROM EXCEL

### ISSUE THREAD

**Issue by IlseFerris** [Hennepin-County/MAXIS-scripts#703]
Jennifer Frey, QI supervisor, is requesting an update to this amazing scripting tool to provide an option to email the worker once the assignment is ready.

**Comment by IlseFerris** [comment]
Jen loves this idea! Let's discuss at our next meeting how to make this happen.

**Comment by C-Love** [comment]
Work is done - has not been used. If there is a bug, we will create a new issue.
